sct-commons/src/test/java/org/lfenergy/compas/sct/commons/dto/FCDAInfoTest.java

```diff
@@ -50,4 +55,68 @@ void testConstructor(){
 
         assertTrue(fcdaInfo1.isValid());
     }
+
+    @ParameterizedTest
+    @MethodSource("providerFCDAInfoObjects")
+    void checkFCDACompatibilitiesForBinding(String testCase, FCDAInfo fcdaInfo, FCDAInfo expectedFcdaInfo) {
```

Comment:
I think it is usefull that tests titles are meaningful. Otherwise you get something like `checkFCDACompatibilitiesForBinding[3] failed` which is not explicit.
And also in the MethodSource `providerFCDAInfoObjects` , each `Argument.of` is explicit otherwise it is hard to understand what is tests. It is better that a `//` comment.